src/green/InterfacesImpl.hpp

```diff
@@ -225,49 +224,48 @@ template <typename StateVariable, typename ODESystem> class Zeta __final {
    */
   void compute(const ProfileEvaluator & eval, const IntegratorParameters & parms) {
     namespace odeint = boost::numeric::odeint;
-    odeint::bulirsch_stoer_dense_out<StateVariable> stepper(
-        parms.eps_abs_, parms.eps_rel_, parms.factor_x_, parms.factor_dxdt_);
+    odeint::runge_kutta4<StateVariable> stepper;
```

Comment:
So it works with a fourth-order Runge-Kutta integrator? Have you tried with the Runge-Kutta-Fehlberg 7(8)?